src/arch/xtensa/include/arch/drivers/interrupt.h

```diff
@@ -15,10 +15,10 @@
 #include <stddef.h>
 #include <stdint.h>
 
-extern char irq_name_level2[];
-extern char irq_name_level3[];
-extern char irq_name_level4[];
-extern char irq_name_level5[];
+extern const char irq_name_level2[];
+extern const char irq_name_level3[];
+extern const char irq_name_level4[];
+extern const char irq_name_level5[];
```

Comment:
I don't get why you are changing a common header and a CAVS-only C file. If these IRQ names aren't valid outside CAVS they should be moved to a platform header file.

If they are in fact common everywhere then why the only instantiation of these that is changed to const is in cAVS?

Reply:
Yes, agree. Added separate commit to move them to cavs platform header.